beangulp/importers/csv.py

```diff
@@ -164,7 +164,7 @@ def prepare_for_identifier(regexps: Union[str, List[str]],
     if isinstance(regexps, str):
         regexps = [regexps]
     matchers = matchers or []
-    matchers.append(('mime', 'text/csv'))
+    matchers.append(('mime', '(text/csv)|(application/vnd\\.ms-excel)'))
```

Comment:
The problem with this change is that the importers based on `beangulp.importers.csv` will start matching all files with the Excel mime type, including xls ans xlsx files. This would be a backward compatibility break, and we should avoid this, especially for an importer base class that has been around for very very long.

What can be done is to add the `mime` matcher only if one has not been passed by the user already. This would allow you to add the matching of the `application/vnd\\.ms-excel` to your importer only. Something like this:
```suggestion
    if not any(m[0] == 'mime' for m in matchers):
        matchers.append(('mime', 'text/csv'))
```
if I remember correctly how this code is supposed to work.